config/settings.yml

```diff
@@ -1169,7 +1169,7 @@
       :orchestration_stack_retired_interval: 10.minutes
       :performance_collection_interval: 3.minutes
       :performance_collection_start_delay: 5.minutes
-      :performance_realtime_purging_interval: 15.minutes
+      :performance_realtime_purging_interval: 25.minutes
```

Comment:
@yrudman yeah, that's a good idea. I wasn't sure if we could keep the current 5 minute gap.